docs/advanced_usage.md

```diff
@@ -221,6 +222,28 @@ the same flag:
 $ notary remove example/collections delegation/path/target --roles=targets/releases
 ```
 
+## Recovering a delegation
+
+It is possible for delegations to get into a state where they delegation file is not
```

Comment:
It might be useful to describe how one can get into such a state in an example.  Like Alice last updated delegation `targets/qa`, but Alice since left the company and an administrator has removed her delegation key from the repo.  Now delegation `targets/qa` has no valid signatures.